Hazel/src/Platform/OpenGL/OpenGLVertexArray.h

```diff
@@ -20,6 +20,7 @@ namespace Hazel {
 		virtual const std::shared_ptr<IndexBuffer>& GetIndexBuffer() const { return m_IndexBuffer; }
 	private:
 		uint32_t m_RendererID;
+		uint32_t m_IndexOffset;
```

Comment:
I feel like this should have a different name such as ``m_VertexBufferIndexOffset``. Generally it should be very clear what it's used for if you were only looking at the header file.